### PR TITLE
Imporve openai api documents

### PR DESCRIPTION
**Pull request by zhaochenyang20**
## Motivation

1. Log server info in cells.
2. add openai API docs reference.
3. link to openai.

## Checklist

- [X] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/contributor_guide.md).
- [X] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/contributor_guide.md).
- [X] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

File: python/sglang/utils.py [hunk 1]
```diff
@@ -300,27 +301,25 @@ def download_and_cache_file(url: str, filename: Optional[str] = None):
     return filename
 
 
-def execute_shell_command(command: str) -> subprocess.Popen:
+def lauch_sglang_server(command: str) -> subprocess.Popen:
```
Comment (merrymercy) [inline python/sglang/utils.py]: call it `execute_shell_command`

BTW `lauch_sglang_server` should be `launch_sglang_server`

File: python/sglang/utils.py [hunk 2]
```diff
@@ -375,3 +374,6 @@ def terminate_process(process):
             torch.cuda.empty_cache()
             torch.cuda.ipc_collect()
         time.sleep(2)
+
+def highlight_text(html_content: str):
```
Comment (merrymercy) [inline python/sglang/utils.py]: `highlight_text` -> `print_highlight`

File: docs/openai_api.ipynb [hunk 3]
```diff
@@ -27,42 +29,96 @@
...
...rained_json_whitespace_pattern=None, log_level='info', log_level_http=None, log_requests=False, show_time_cost=False, api_key=None, file_storage_pth='SGLang_storage', enable_cache_report=False, watchdog_timeout=600, dp_size=1, load_balance_method='round_robin', dist_init_addr=None, nnodes=1, node_rank=0, json_model_override_args='{}', enable_double_sparsity=False, ds_channel_config_path=None, ds_heavy_channel_num=32, ds_heavy_token_num=256, ds_heavy_channel_type='qk', ds_sparse_decode_threshold=4096, lora_paths=None, max_loras_per_batch=8, attention_backend='flashinfer', sampling_backend='flashinfer', grammar_backend='outlines', disable_flashinfer=False, disable_flashinfer_sampling=False, disable_radix_cache=False, disable_regex_jump_forward=False, disable_cuda_graph=False, disable_cuda_graph_padding=False, disable_disk_cache=False, disable_custom_all_reduce=False, disable_mla=False, disable_penalizer=False, disable_nan_detection=False, enable_overlap_schedule=False, enable_mixed_chunk=False, enable_torch_compile=False, torch_compile_max_bs=32, cuda_graph_max_bs=160, torchao_config='', enable_p2p_check=False, triton_attention_reduce_in_fp32=False, num_continuous_decode_steps=1)\n",
```
Comment (merrymercy) [inline docs/openai_api.ipynb:41]: Is it possible to put a gray background on the server output or make the font in gray?

File: docs/openai_api.ipynb [hunk 4]
```diff
@@ -27,42 +29,96 @@
...
-    "--port 30000 --host 0.0.0.0 --log-level warning\n",
-    "\"\"\"\n",
+    "server_process = lauch_sglang_server(\n",
+    "    command=\"python -m sglang.launch_server --model-path meta-llama/Meta-Llama-3.1-8B-Instruct --port 30000 --host 0.0.0.0\"\n",
     ")\n",
     "\n",
     "wait_for_server(\"http://localhost:30000\")\n",
-    "print(\"Server is ready. Proceeding with the next steps.\")"
+    "\n",
+    "highlight_text(\"Server is ready. Proceeding with the next steps.\")"
```
Comment (merrymercy) [inline docs/openai_api.ipynb]: - mention that the cell will contain both the server output and notebook outout.
- mention that typically the server will run on a separate terminal, but here we combine the output into a single cell to demonstrate the usage better. Mention that the server output will be in gray while the notebook output will be highlighted.

**Review by merrymercy — CHANGES_REQUESTED** [review]

File: docs/openai_api.ipynb (hunk 3 above)
Comment (zhaochenyang20) [reply docs/openai_api.ipynb:41]: I attempted to modify the CSS but found it to be ineffective (a lot of nbsphinx users also shared this concern). Apologies for this. Do you consider this extremely necessary? At the very least, the current style is functional for me. Maybe we can change the color of our highlights?

image

File: docs/conf.py [hunk 5]
```diff
@@ -127,3 +130,14 @@
...
 
 html_theme = "sphinx_book_theme"
+
+
+nbsphinx_prolog = """
+.. raw:: html
+
+    <style>
+        .output_area.stderr, .output_area.stdout {
+            color: #d3d3d3 !important; /* 浅灰色 */
```
Comment (merrymercy) [inline docs/conf.py]:
```suggestion
            color: #d3d3d3 !important; /* light gray */
```

**Pull request merged**